packages/react-native-views/src/components/CameraView/index.jsx

```diff
@@ -1,18 +1,43 @@
-import React, { useCallback, useEffect, useRef, useState } from 'react';
-
-import noop from 'lodash.noop';
-
+import React, { useMemo, useState } from 'react';
+import { Platform, SafeAreaView, StatusBar, StyleSheet, View } from 'react-native';
 import Components, { propTypes } from '@monkvision/react-native';
 import { Sight, values } from '@monkvision/corejs';
+import noop from 'lodash.noop';
 
-import { View, Platform, SafeAreaView, StatusBar } from 'react-native';
-import { FAB, Snackbar, Text, useTheme, Modal } from 'react-native-paper';
+import useFakeActivity from './hooks/useFakeActivity';
+import usePictures from './hooks/usePictures';
+import useSuccess from './hooks/useSuccess';
+import useUI from './hooks/useUI';
 
 import ActivityIndicatorView from '../ActivityIndicatorView';
-import AdvicesView from '../AdvicesView';
 
-import useSights from './useSights';
-import styles from './styles';
+import CameraControls from './CameraControls';
+import CameraOverlay from './CameraOverlay';
+import CameraPopUps from './CameraPopUps';
+import CameraScrollView from './CameraScrollView';
+
+const styles = StyleSheet.create({
+  root: {
+    backgroundColor: '#000',
+    ...Platform.select({
+      native: { flex: 1 },
+      default: { display: 'flex', flex: 1, height: '100vh' },
+    }),
+  },
+  container: {
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    overflow: 'hidden',
+    minWidth: '100%',
+    minHeight: '100%',
+    backgroundColor: '#000',
+    justifyContent: 'space-between',
+    ...Platform.select({
+      native: { flex: 1 },
+      default: { display: 'flex', flex: 1 },
+    }),
+  },
+});
 
 const SIDEBAR_WIDTH = 250;
 const makeRatio = (width, height) => `${(width - SIDEBAR_WIDTH) / 240}:${height / 240}`;
```

Comment:
Why not put the SIDEBAR_WIDTH in the style (and make `make_ratio` have it as a parameter)

The hardcoded 240 seems fishy :suspect:

Reply:
This is not related to this PR. Maybe create a question issue ?

Reply:
:sad_panda: